operator/builtin/output/googlecloud/operator.go

```diff
@@ -73,12 +73,24 @@ func (g *GoogleCloudOutput) Stop() error {
 	g.flusher.Stop()
 	g.Debug("Stopped flusher")
 
-	// TODO handle buffer close entries
-	if _, err := g.buffer.Close(); err != nil {
-		return fmt.Errorf("failed to close buffer: %w", err)
+	entries, err := g.buffer.Close()
+	if err != nil {
+		g.Errorf("Failed to retreive entries: %w", err)
```

Comment:
This was returning a wrapped error before we should make sure it still does. I don't think we need a log message here as it'll get bubbled up.